eng/pipelines/aggregate-reports.yml

```diff
@@ -128,16 +128,15 @@ jobs:
         displayName: 'Run CredScan'
         inputs:
           suppressionsFile: 'eng\CredScanSuppression.json'
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
```

Comment:
Why would you publish before the a analysis?

Reply:
I am thinking to fail the post analysis if new issues coming. Then the publish step needs to either put before post analysis or run under condition of `successOrFailed`. Both two do not make too much difference, so I rearrange the order which is not blocking the publish step.

Reply:
That's fair. I agree it doesn't matter for our cases the order just seemed a little odd.